core/deployment/src/main/java/io/quarkus/deployment/util/ExecUtil.java

```diff
@@ -0,0 +1,120 @@
+
+package io.quarkus.deployment.util;
+
+import java.io.BufferedReader;
+import java.io.File;
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.InputStreamReader;
+import java.util.function.Function;
+
+public class ExecUtil {
+
+    private static final Function<InputStream, Runnable> PRINT_OUTPUT = i -> new PrintOutput(i);
+    private static final Function<InputStream, Runnable> SILENT = i -> new Runnable() {
```

Comment:
This will fail if the process produces a lot of output. You need to consume the output otherwise the process will hang once the output buffer fill up (output buffer size is system dependent).

Reply:
Thanks for the advice. Will apply that soon.